security-oauth2/src/main/java/io/micronaut/security/oauth2/client/DefaultOpenIdProviderMetadata.java

```diff
@@ -34,48 +33,125 @@
  */
 @ReflectiveAccess
 @Serdeable
-@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
```

Comment:
serde supports this, this change shouldn't be necessary

Reply:
I just wanted to remove even the compile time dependency to mn.jackson.databind

Reply:
why? it should be fine as a compile-time dep, no?

Reply:
I don't see much benefit on `JsonNaming` + the extra `compileOnly` and prefer the the explicit `@JsonProperty`. The only class which really benefits from `JsonNaming` is `DefaultOpenIdProviderMetadata`.

Nonetheless, I have reverted that change here: https://github.com/micronaut-projects/micronaut-security/pull/1395/commits/c5637c6e9e7222e3edfa1dafcf1b5bd3c1ec0b65

Reply:
it's your repo, do whatever you think is better. the changes LGTM.